sips/sip-x/sip-x-authentication-protocol.md

```diff
@@ -111,10 +151,9 @@ https://github.com/PravicaInc/wise-js
 
 ## Authenticators
 
-
 ### Hiro Wallet
 
-Hiro Wallet is an wallet that handles authentication and transaction signing.
+Hiro Wallet is an browser extension that handles authentication and transaction signing.
```

Comment:
Hiro Wallet is **a** browser extension...

Reply:
@314159265359879 Thank you fore reading the sip and finding these. Fixed